mcs/class/corlib/System.Runtime.InteropServices.RuntimeInformation/RuntimeInformation.cs

```diff
@@ -35,6 +35,9 @@ namespace System.Runtime.InteropServices
 {
 	public static class RuntimeInformation
 	{
+		[DllImport("__Internal", CharSet = CharSet.Auto)]
```

Comment:
Just remembered that we can't use `__Internal` as it would end up not working in embedded cases like Xamarin.Android because it searches for the method in the host which embeds Mono.

You'll need to add a new icall for this.